File: VPet-Speaking/GetMessage.cs
namespace VPet.Plugin.Speaking
{
    /// <summary>
    /// DIY「说话」调试用固定文本（合成播放，不调用 LLM）。
    /// </summary>
    public static class GetMessage
    {
        /// <summary>点击「说话」时固定合成的调试文本。</summary>
        public const string ChatPrompt = "好无聊啊，和我聊聊天吧";

        /// <summary>兼容旧命名。</summary>
        public const string TestSample = ChatPrompt;

        public static string get_message(string? message)
        {
            try
            {
                if (message != null)
                    return message;
            }
            catch
            {
                // ignore
            }
            return string.Empty;
        }
    }
}

File: VPet-Speaking/get_message.py
# 测试样例：获取要合成的文本
# 对应 C# 侧 GetMessage.get_message / GetMessage.TestSample

def get_message(message):
    try:
        if message is not None:
            return message
    except Exception as e:
        return str(e)
    return ""


# 测试用例（短句，便于本地 F5 低延迟合成）：
message = get_message("你好，我是太乙真人。")
